packages/@okta/vuepress-site/docs/guides/set-up-self-service-registration/overview/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Overview
+---
+
+A new user's first impression starts with the registration process, making it a critical aspect to get right. Because you're asking a user to give you information for the first time, you need to personalize and streamline the experience.
+
+Okta's self-service registration lets you configure a custom app or the Okta Homepage for use when users self-register. After you configure and enable your self-service registration policy, a sign-up link appears in the Okta Sign-In Widget. Users who click this link are directed to a Create Account registration form that is based on your registration policy.
+
+**Self-service registration supports these registration workflows:**
+
+* **Make email verification mandatory.** After registering their information, users are immediately sent an email to verify their email address. Users must click the link in the email to complete the registration process. Users are then redirected to your app or to your org's Okta Homepage.
+
+> **Note:** The lifetime of the emailed link is dictated by the **Activation emails are valid for** setting on the **Security** > **General** page of the Admin Console.
+
+* **Make email verification optional.** After registration, end users are immediately redirected to your custom app/portal or to your org's Okta Homepage. Okta then sends the user an email requesting verification of their email address.
+
+## Before you begin
+
+> **Note:** This is an Early Access feature. To enable it, contact [Okta Support](https://support.okta.com/help/s/?_ga=2.17747641.1660906902.1597076228-1076744453.1575496867).
+
+* Super admin permissions are required to enable self-service registration.
```

Comment:
This might be confusing to someone that didn't create the org, how do they know if they have permissions? (i have no suggestions on how to improve this though 🤷 )